pkg/sentry/pgalloc/pgalloc.go

```diff
@@ -1095,6 +1097,16 @@ func (f *MemoryFile) findReclaimable() (platform.FileRange, bool) {
 		for seg := f.usage.LowerBoundSegment(f.minReclaimablePage); seg.Ok(); seg = seg.NextSegment() {
 			if seg.ValuePtr().refs == 0 {
 				f.minReclaimablePage = seg.End()
+				if f.reclaimableSize > seg.Range().Length() {
+					f.reclaimableSize -= seg.Range().Length()
+				} else {
+					f.reclaimableSize = 0
+				}
+				// Stop current reclaim, wait for next reclaim.
+				if f.reclaimableSize == 0 {
+					f.reclaimable = false
```

Comment:
Can we just replace f.reclaimable with f.reclaimableSize entirely, removing the need for this branch?